lib/AArch64Base.ml

```diff
@@ -33,7 +36,7 @@ type gpr =
   | R16 | R17 | R18 | R19
   | R20 | R21 | R22 | R23
   | R24 | R25 | R26 | R27
-  | R28 | R29 | R30
+  | R28 | R29 | R30 | RSpecial
```

Comment:
Why not calling the register SP ?

Reply:
I can't remember why I did this, this commit is old, will fix